openmetadata-spec/src/main/resources/json/schema/entity/teams/teamHierarchy.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/teams/teamHierarchy.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Team Hierarchy",
+  "description": "This schema defines the Team entity with Hierarchy. Hierarchical teams are supported `Organization` -> `BusinessUnit` -> `Division` -> `Department` -> `Group`.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.entity.teams.TeamHierarchy",
+  "definitions": {
+    "teamHierarchyList": {
+      "type": "array",
+      "items": {
+        "$ref": "teamHierarchy.json"
+      },
+      "default": null
+    }
+  },
+  "properties": {
+    "id": {
+      "$ref": "../../type/basic.json#/definitions/uuid"
+    },
+    "teamType": {
+      "description": "Team type",
+      "$ref": "../../entity/teams/team.json#/definitions/teamType"
+    },
+    "name": {
+      "description": "A unique name of the team typically the team ID from an identity provider. Example - group Id from LDAP.",
+      "$ref": "../../type/basic.json#/definitions/entityName"
+    },
+    "fullyQualifiedName": {
+      "description": "FullyQualifiedName same as `name`.",
+      "$ref": "../../type/basic.json#/definitions/fullyQualifiedEntityName"
+    },
+    "displayName": {
+      "description": "Name used for display purposes. Example 'Data Science team'.",
+      "type": "string"
+    },
+    "href": {
+      "description": "Link to the resource corresponding to this entity.",
+      "$ref": "../../type/basic.json#/definitions/href"
+    },
+    "children" : {
```

Comment:
we can just add this to team.json right, any reason to create another class for this?